pages/blog.js

```diff
@@ -9,12 +9,27 @@ import 'isomorphic-fetch';
 import Layout from '../components/layout/main';
 import Posts from '../components/blog/posts';
 import Background from '../components/content/background';
+import TagsFilter from '../components/tags-filter/TagsFilter';
+
+const flatten = deepArray => deepArray.reduce((a, b) => a.concat(b), []);
```

Comment:
Do we really need "transformateCategories" function?
Is there any possibility to choose two identical tags?
Is there any possibility to select a tag that not exist?

Website functionality doesn't change if I replace line #56 with:
`? chosenCategory.split(',')`